INSTALL.rst.txt

```diff
@@ -61,6 +61,8 @@ To perform an inplace build that can be run from the source folder run::
 
     python setup.py build_ext --inplace -j 4
 
+Note that, due to the way most Linux distributions are handling the Python 3 migration, Linux users may have to replace the ``python`` command with ``python3``. See `Requirements for Installing Packages <https://packaging.python.org/tutorials/installing-packages/>`_ for more details.
```

Comment:
I think this should say that this command will on these installations install on python 2, and to use `python3` to install on python 3.

Reply:
Shouldn't the instructions tell people to use pip?

Reply:
@pv - Basic installation already covers pip. This was for building from source :).
@eric-wieser - Making that edit now, will make a pull request today!